docs/pages/zkvm/cli-quick-start.mdx

```diff
@@ -41,6 +41,7 @@ This should print the available CLI commands.
 
 ### 2. Create a new Nexus project
 
+Somewhere out of the nexus-zkvm checkout:
```

Comment:
Maybe slightly better wording could be:

`Somewhere out of the nexus-zkvm checkout, run:`

Reply:
updated both occurrences.